src/main.js

```diff
@@ -1,10 +1,18 @@
 import Vue from 'vue';
-import App from './App.vue';
-import router from './router';
-import store from './store';
+import * as Sentry from '@sentry/browser'; // https://docs.sentry.io/platforms/javascript/vue/
+import { Vue as VueIntegration } from '@sentry/integrations';
 import 'bootstrap/dist/css/bootstrap.css'; // eslint-disable-line
 import 'bootstrap-vue/dist/bootstrap-vue.css'; // eslint-disable-line
 import { BootstrapVue, IconsPlugin } from 'bootstrap-vue'; // eslint-disable-line
+import App from './App.vue';
+import router from './router';
+import store from './store';
+
+
+Sentry.init({
+  dsn: 'https://521cd3a837e643f687d530e7ac249b4d@o433473.ingest.sentry.io/5388723',
```

Comment:
What is this for? Sentry is frreeee, right?

Reply:
This is an identifier for our application